openstack_sdk/src/api/image/v2/metadef/namespace/resource_type/create.rs

```diff
@@ -34,7 +34,7 @@ use std::borrow::Cow;
 pub struct Request<'a> {
     /// Resource type names should be aligned with Heat resource types whenever
     /// possible:
-    /// https://docs.openstack.org/heat/latest/template_guide/openstack.html
+    /// <https://docs.openstack.org/heat/latest/template_guide/openstack.html>
```

Comment:
As the file header states this file is auto-generated. It makes no sense to fix it here since it will be reset in next re-generation and it also causes bigger diffs in the codegenerator. I know it is annoying, but we can not easily fix content coming from OpenStack projects. Please undo this change

Reply:
Ups, didn't look much around. I only blindly followed the warnings of `cargo doc` :/